src/applications/mhv-medical-records/containers/Allergies.jsx

```diff
@@ -249,40 +249,43 @@ ${allergies.map(entry => generateAllergyListItemTxt(entry)).join('')}`;
               }}
             />
           )}
-        {isLoadingAcceleratedData || isLoading ? (
+        {(isLoadingAcceleratedData || isLoading) && (
           <div className="vads-u-margin-y--8">
             <TrackedSpinner
               id="allergies-page-spinner"
-              message="We’re loading your records."
+              message="We're loading your records."
```

Comment:
This seems fine, though when you put a single-quote into JSX directly, you get this linting error.

<img width="707" height="102" alt="Image" src="https://github.com/user-attachments/assets/2de0a896-261f-488b-afae-33076cb8b0cf" />